package-lock.json

```diff
@@ -0,0 +1,4081 @@
+{
```

Comment:
i'm not sure about committing `package-lock.json` or `yarn.lock` for a library repo, it adds git bloat and doesn't really provide any benefit when libraries are developed sparingly and you want the dependencies to match what a new user would experience (not what's in your lock file). thoughts?